slices/php-common.yaml

```diff
@@ -0,0 +1,49 @@
+package: php-common
+
+essential:
+  - php-common_copyright
+
+slices:
+
+  bins:
+    essential:
+      - php-common_libs
+      - php-common_registry
+    contents:
+      /usr/sbin/phpdismod:
+      /usr/sbin/phpenmod:
+      /usr/sbin/phpquery:
+
+  libs:
+    contents:
+      /usr/lib/php/php-fpm-socket-helper:
+      /usr/lib/php/php-helper:
+      /usr/lib/php/sessionclean:
+
+  registry:
+    # This slice is only required for phpquery. When phpenmod or
+    # phpdismod are run with the -r flag. A new empty file is
+    # created in the registry location. These files are used by
+    # phpquery to track the modules currently enabled.
+    contents:
+      /var/lib/php/modules/8.3/registry/calendar: { text: "" }
+      /var/lib/php/modules/8.3/registry/ctype: { text: "" }
+      /var/lib/php/modules/8.3/registry/exif: { text: "" }
+      /var/lib/php/modules/8.3/registry/ffi: { text: "" }
+      /var/lib/php/modules/8.3/registry/fileinfo: { text: "" }
+      /var/lib/php/modules/8.3/registry/ftp: { text: "" }
+      /var/lib/php/modules/8.3/registry/gettext: { text: "" }
+      /var/lib/php/modules/8.3/registry/iconv: { text: "" }
+      /var/lib/php/modules/8.3/registry/pdo: { text: "" }
+      /var/lib/php/modules/8.3/registry/phar: { text: "" }
+      /var/lib/php/modules/8.3/registry/posix: { text: "" }
+      /var/lib/php/modules/8.3/registry/shmop: { text: "" }
+      /var/lib/php/modules/8.3/registry/sockets: { text: "" }
+      /var/lib/php/modules/8.3/registry/sysvmsg: { text: "" }
+      /var/lib/php/modules/8.3/registry/sysvsem: { text: "" }
+      /var/lib/php/modules/8.3/registry/sysvshm: { text: "" }
+      /var/lib/php/modules/8.3/registry/tokenizer: { text: "" }
```

Comment:
What about `/var/lib/php/sessions/`?
Should it be added to one of these slice definitions?